modules/simple_forms_api/app/services/simple_forms_api/notification_email.rb

```diff
@@ -21,10 +21,11 @@ class NotificationEmail
     }.freeze
     SUPPORTED_FORMS = TEMPLATE_IDS.keys
 
-    def initialize(form_data:, form_number:, confirmation_number:, notification_type: :confirmation, user: nil)
-      @form_data = form_data
-      @form_number = form_number
-      @confirmation_number = confirmation_number
+    def initialize(config, notification_type: :confirmation, user: nil)
+      @form_data = config[:form_data]
+      @form_number = config[:form_number]
+      @confirmation_number = config[:confirmation_number]
```

Comment:
The consolidation is nice and necessary given our linter settings, but we need some sort of validation to ensure the initialization will fail gracefully and clearly when one or multiple of these parameters are not provided.